.classpath

```diff
@@ -2,7 +2,7 @@
 <classpath>
 	<classpathentry kind="lib" path="src/test/resources/lib/javamail/mail.jar"/>
 	<classpathentry kind="con" path="com.google.appengine.eclipse.core.GAE_CONTAINER"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry exported="true" kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
```

Comment:
The changes that are not related to the issue should not be part of the pull request. You should remove them.